addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -61,6 +61,7 @@ def send_mail_test(self):
                 'attachment_ids': [(4, attachment.id) for attachment in mailing.attachment_ids],
                 'auto_delete': False,  # they are manually deleted after notifying the document
                 'mail_server_id': mailing.mail_server_id.id,
+                'res_id': self.env.user.id,
```

Comment:
can you also add a small test ?

Reply:
I added a new part in an already existing test in test_mass_mailing with these specific cases.